_posts/2018-03-15-crossdeploying-device-images-to-a-different-account.adoc

```diff
@@ -5,25 +5,26 @@
 :page-order: 3
 :icons: font
 :sectnums:
-:garage-deploy-version: 2018.8
+:garage-deploy-version: 2018.9
 
 For our recommended production workflow, you will need to move device images from one account to another from time to time--for example, sending a development build you're happy with to the QA team, or sending that build to the deployment team once it's passed QA. We provide a tool called `garage-deploy` for doing just that.
 
 . Install the `garage-deploy` tool
 +
 NOTE: If you've link:../prod/rotating-signing-keys.html[rotated your signing keys] already, `garage-deploy` will already be installed, and you can skip this step.
 +
-On a Ubuntu 16.04 machine, download the `garage-deploy` package and install it:
+We currently provide released versions of `garage-deploy` for Ubuntu 18.04 (Bionic) and Ubuntu 16.04 (Xenial) which are available on https://github.com/advancedtelematic/aktualizr/releases/tag/{garage-deploy-version}.
+On a Ubuntu 18.04 machine, download the `garage-deploy` package and install it:
 +
 [subs="attributes"]
 ----
-wget https://github.com/advancedtelematic/aktualizr/releases/download/{garage-deploy-version}/garage_deploy.deb
-sudo apt install ./garage_deploy.deb
+wget https://github.com/advancedtelematic/aktualizr/releases/download/{garage-deploy-version}/garage_deploy-ubuntu_18.04.deb
+sudo apt install ./garage_deploy-ubuntu_18.04.deb
 ----
 +
 [TIP]
 ====
-This package is built against Ubuntu 16.04 and may work for other distributions, but if it doesn't work for your setup, you can checkout https://github.com/advancedtelematic/aktualizr/tree/{garage-deploy-version}[aktualizr] and build the deb package or binary yourself:
+These packages are specific to the distributions they were built for. If yours is not in the list, you can checkout https://github.com/advancedtelematic/aktualizr/tree/{garage-deploy-version}[aktualizr], install the required dependencies link:https://github.com/advancedtelematic/aktualizr/tree/2018.9#dependencies[listed here] (exact package names may vary) and build the deb package yourself:
```

Comment:
In the second link, please use {garage-deploy-version} in place of 2018.9.

Reply:
Thanks! Fixed (force pushed)